packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -714,6 +719,25 @@ export class DetailsList extends BaseComponent<IDetailsListProps, IDetailsListSt
       onActiveItemChanged(items[index], index, ev);
     }
   }
+
+  private _getItemKey(item: any, itemIndex: number): string | number {
+    const { getKey } = this.props;
+
+    let itemKey: string | number | undefined = undefined;
+    if (item) {
+      itemKey = item.key;
+    }
+
+    if (getKey) {
```

Comment:
nit comment to update the props to say these keys are used for focus as well as that they must be unique

Reply:
It already does:
```
/** Optional callback to get the item key, to be used in the selection and on render. */
  getKey?: (item: any, index?: number) => string;
```

I think uniqueness requirement for keys is well known in React land.